cmd/minikube/cmd/mount.go

```diff
@@ -46,6 +50,14 @@ var mountCmd = &cobra.Command{
 			fmt.Fprintln(os.Stderr, errText)
 			os.Exit(1)
 		}
+		if args[0] == "kill" {
```

Comment:
I understand why you did this, but its a little weird to treat it as a subcommand without it actually using cobra.  It's probably not worth it to change the API, but adding a `mount add` and a `mount kill` would fix this (maybe even a `mount list`).